syntax = "proto3";

package qoin;

import "qoin/proto/landmark.proto";

service FaceMesh {
  rpc FaceMeshStream(FaceMeshRequest) returns (stream FaceMeshReply) {}
}

message FaceMeshRequest {}

message FaceMeshReply { repeated LandmarkList landmark_list = 1; }
